packages/hash/realtime/package.json

```diff
@@ -18,13 +18,13 @@
   },
   "dependencies": {
     "@hashintel/hash-backend-utils": "0.0.0",
-    "slonik": "24.1.2"
+    "slonik": "24.1.2",
+    "typescript": "4.6.2"
```

Comment:
A bundling step could let us move `typescript` back into DevDeps. Currently we need it to make ts-node happy

Reply:
Yeah if we use `ts-node` in the production container, it would make sense to explicitly move it to `dependencies`. Even if the result is the same because of some transient deps, we communicate what we do better and potentially avoid mission dependencies in the future after `yarn upgrade`.

Reply:
I guess that `ts-node-dev` still remains in `devDependencies`. We only need it for `yarn ... dev`, which is not called inside the containers.